test/integration/data/flu-metrocast/hub-config/admin.json

```diff
@@ -0,0 +1,24 @@
+{
+    "schema_version": "https://raw.githubusercontent.com/hubverse-org/schemas/main/v5.0.0/admin-schema.json",
+    "name": "Flu MetroCast",
+    "maintainer": "epiENGAGE",
+    "contact": {
+        "name": "Dongah Kim",
+        "email": "donga0223@gmail.com"
```

Comment:
We should avoid adding personal details of folks in our software test materials even if it does appear in public hubs

Reply:
Yeah, they've often slipped through but we try to remove them wherever we can. Am sure hubData could do with another pass at some point. It's mainly for privacy reasons.